rafal_szyinski/pin_controls.h

```diff
@@ -0,0 +1,65 @@
+#ifndef __PIN_CONTROLS_H__
+#define __PIN_CONTROLS_H__
+
+#include "pins.h"
+
+void turn_off_color();
+void turn_on_off();
+void change_mode();
+void disco();
+
+void turn_off_color()
+{
+	digitalWrite(red_pin, LOW);  	
+	digitalWrite(blue_pin, LOW);
+	digitalWrite(green_pin, LOW);
+}
+
+void turn_on_off()
+{
+	switch(state)
```

Comment:
Just change state in _main.cpp_

Reply:
so you want that one line if statement that change state was in main.cpp for `turn_on_off` and for `change_mode`?